controller/tests/controller_server/utils_test.py

```diff
@@ -61,12 +62,6 @@ def test_validate_volume_capabilities(self):
         with self.assertRaises(ValidationException):
             utils.validate_csi_volume_capabilties([caps])
 
-        caps = Mock()
-        caps.mount = None
-        caps.access_mode.mode = access_types.SINGLE_NODE_READER_ONLY
-        with self.assertRaises(ValidationException):
```

Comment:
why is this removed?

Reply:
mount can never be none
The parallel test is added to check block. Which makes HasField return false instead of setting mount to None